python/lib/dependabot/python/language_version_manager.rb

```diff
@@ -33,8 +33,12 @@ def install_required_python
       end
 
       def python_major_minor
-        @python ||= Python::Version.new(python_version)
-        "#{@python.segments[0]}.#{@python.segments[1]}"
+        @python_major_minor ||= self.class.truncate_version_to_major_minor(python_version)
```

Comment:
Does anything else rely on this method? This is changing the return value from a string to `Python::Version`

Reply:
True, I could slap a `.to_s` on the end, but I audited all the usages of it and from what I can tell they should all be fine with duck typing... ie, they're called from within a string, or they're used as values in a hash that is passed to `TomlRB.dump()` which I expect will call `.to_s` on it.

So I think we'll be fine with retaining the typing here and only coercing to string at the point of use.

The primary drawback I see of this is that it potentially ties us tighter to the `Gem::Version` class, when in many ways we'd rather treat the versions as opaque strings wherever possible for passing to native helpers. That could be a reasonable argument for having `python_major_minor` continue to return a string.

Thoughts?

Reply:
Got it! No strong preference, just wanted to make sure it was expected & works.